client/src/app/tabs/dmn/modeler/features/decision-table-keyboard-bindings/DecisionTableKeyboardBindings.js

```diff
@@ -0,0 +1,90 @@
+import {
+  isCmd as isCommandOrControl,
+  isKey,
+  isShift
+} from 'diagram-js/lib/features/keyboard/KeyboardUtil';
+
+import {
+  findSelectableAncestor
+} from 'dmn-js-decision-table/lib/features/cell-selection/CellSelectionUtil';
+
+export default class DecisionTableKeyboardBindings {
+  constructor(decisionTable, commandStack, editorActions, eventBus) {
+    this._decisionTable = decisionTable;
+    this._commandStack = commandStack;
+    this._editorActions = editorActions;
+    this._eventBus = eventBus;
+
+    eventBus.on('attach', this._addEventListeners);
+
+    eventBus.on('detach', this._removeEventListeners);
+  }
+
+    _addEventListeners = () => {
+      const container = this._getContainer();
+
+      container.addEventListener('keydown', this.handleKeydown, true);
+    }
+
+    _removeEventListeners = () => {
+      const container = this._getContainer();
+
+      container.removeEventListener('keydown', this.handleKeydown);
+    }
+
+    handleKeydown = event => {
+      const commandStack = this._commandStack;
+      const editorActions = this._editorActions;
+
+      if (isUndo(event)) {
+        commandStack.canUndo() && commandStack.undo();
+
+        event.preventDefault();
+      }
+
+      if (isRedo(event)) {
+        commandStack.canRedo() && commandStack.redo();
+
+        event.preventDefault();
+      }
+
+      if (isSelectCell(event)) {
+        if (!this._hasSelectableAncestor(event.target)) {
+          return;
+        }
+
+        const cmd = isShift(event) ? 'selectCellAbove' : 'selectCellBelow';
+
+        editorActions.trigger(cmd);
+
+        event.preventDefault();
+        event.stopPropagation();
```

Comment:
Can you explain why we stop propagation only for cell selection event?

Reply:
I ran in strange behavior before adding this, in a way of the [line-break handler](https://github.com/bpmn-io/dmn-js/blob/master/packages/dmn-js-shared/src/components/ContentEditable.js#L135) also got triggered after the cell was selected. Therefore it always added a new line into the next selected cell.